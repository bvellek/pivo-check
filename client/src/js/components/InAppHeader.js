import React from 'react';
import { Link } from 'react-router-dom';
import SVGLogo from './SVGLogo';
import Logout from './Logout';

const InAppHeader = () => (
  <header className="in-app-header top-header" role="banner">
    <div className="in-app-logo">
      <span className="visually-hidden">Pivo-Check</span>
      <SVGLogo />
    </div>
    <nav role="navigation">
      <Link to='/cities'>My Cities</Link>
      <Logout />
    </nav>
  </header>
);

export default InAppHeader;
