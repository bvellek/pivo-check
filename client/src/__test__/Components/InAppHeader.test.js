import React from 'react';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';

import InAppHeader from '../../js/components/InAppHeader';

describe('<InAppHeader />', () => {
  it('it renders correctly', () => {
    const component = shallow(
      <InAppHeader />
    );
    expect(shallowToJson(component)).toMatchSnapshot();
  });
});
