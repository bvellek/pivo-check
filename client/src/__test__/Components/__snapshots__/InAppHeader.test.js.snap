// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<InAppHeader /> it renders correctly 1`] = `
<header
  className="in-app-header top-header"
  role="banner"
>
  <div
    className="in-app-logo"
  >
    <span
      className="visually-hidden"
    >
      Pivo-Check
    </span>
    <SVGLogo />
  </div>
  <nav
    role="navigation"
  >
    <Link
      replace={false}
      to="/cities"
    >
      My Cities
    </Link>
    <withRouter() />
  </nav>
</header>
`;
